crates/network-scanner/Cargo.toml

```diff
@@ -8,9 +8,12 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.79"
+get_if_addrs = "0.5.3"
```

Comment:
issue: This crate is unmaintained since 2019 (the repository has been archived).

question: Did you consider this? https://lib.rs/crates/if-addrs
This is maintained fork.